routes/images.js

```diff
@@ -40,21 +41,21 @@ router.delete(
   ImageController.destroy
 );
 
-// Batch edit images.
-router.patch(
-  "/",
-  celebrate({
-    body: {
-      images: Joi.array()
-        .items(
-          Joi.object({
-            id: Joi.string().required(),
-            s3key: Joi.string().required()
-          })
-        )
-        .min(1)
-    }
-  }),
+// Uppdate an Image: If I turn celebrate on, the edit function won't work. I need to learn how to use celebrate :(
```

Comment:
I don't know how to use celebrate when editing image.
Will ask you tmr.

Reply:
ok